### PR TITLE
[Modular] Adds emotes from other servers

### PR DESCRIPTION
**Pull request by ReturnToZender**
## About The Pull Request
Ports a bunch of emotes from other servers. May have some redundancies, I tried to remove most redundant emotes, unless they had more sounds. Suggestions welcome!

## Changelog

:cl:
Adds a bunch of... less low-roleplay emotes from CitRP and SPLURT. Suggestions welcome!
/:cl:

### CODE REVIEW

File: modular_zubbers/code/datums/emotes.dm [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+	key = "squint"
+	key_third_person = "squints"
+	message = "squints their eyes." // i dumb
+	emote_type = EMOTE_VISIBLE
+
+/datum/emote/living/mlem
+	key = "mlem"
+	key_third_person = "mlems"
+	message = "sticks their tongue for a moment. Mlem!"
+	emote_type = EMOTE_VISIBLE
```
Comment (Cyprex) [inline modular_zubbers/code/datums/emotes.dm:17]: LRP

File: modular_zubbers/code/datums/emotes.dm [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+	vary = TRUE
+	sound = 'modular_zubbers/sound/voice/moth/mothchitter2.ogg'
+
+/datum/emote/living/audio/boowomp
+	key = "boowomp"
+	key_third_person = "boowomps"
+	message = "produces a sad boowomp."
+	message_mime = "produces a silent boowomp."
+	vary = TRUE
+	sound = 'modular_zubbers/sound/voice/boowomp.ogg'
```
Comment (Cyprex) [inline modular_zubbers/code/datums/emotes.dm:57]: LRP

File: modular_zubbers/code/datums/emotes.dm [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+/datum/emote/living/audio/eyebrow2
+	key = "eyebrow2"
+	key_third_person = "eyebrows2"
+	message = "<b>raises an eyebrow.</b>"
+	message_mime = "<b>raises an eyebrow with quaking force!</b>"
+	vary = TRUE
+	sound = 'modular_zubbers/sound/voice/vineboom.ogg'
```
Comment (Cyprex) [inline modular_zubbers/code/datums/emotes.dm:65]: LRP

File: modular_zubbers/code/datums/emotes.dm [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+	key_third_person = "teshscreams"
+	message = "screams like a chicken!"
+	message_mime = "acts out a chicken scream."
+	vary = TRUE
+	sound = 'modular_zubbers/sound/voice/teshari/teshariscream.ogg'
+
+/datum/emote/living/audio/teshari/sneeze
+	key = "teshsneeze"
+	key_third_person = "teshsneeze"
+	message = "sneezes cutely."
```
Comment (Cyprex) [inline modular_zubbers/code/datums/emotes.dm:178]: I need a Teshari expert on this- how the heck do these little buggers sneeze "cutely" @projectkepler-ru

File: modular_zubbers/code/datums/emotes.dm (hunk 1 above)
Comment (ReturnToZender) [reply modular_zubbers/code/datums/emotes.dm:17]: The-- the tongue mlem?

File: modular_zubbers/code/datums/emotes.dm (hunk 3 above)
Comment (ReturnToZender) [reply modular_zubbers/code/datums/emotes.dm:65]: yeah that's fair

File: modular_zubbers/code/datums/emotes.dm (hunk 2 above)
Comment (ReturnToZender) [reply modular_zubbers/code/datums/emotes.dm:57]: aww, I loved boowomp :( can get rid of it though

File: modular_zubbers/code/datums/emotes.dm (hunk 4 above)
Comment (ReturnToZender) [reply modular_zubbers/code/datums/emotes.dm:178]: I dunno if I'd consider Kepler a "Teshari expert"

**Comment by nevimer** [comment]
Update??

**Review by BurgerLUA — APPROVED** [review]
Some of these are kind of LRP (but funny) but there are no code issues.

**Pull request closed without merging**